Blockzilla/OverlayView.swift

```diff
@@ -27,17 +29,21 @@ class OverlayView: UIView {
         super.init(frame: CGRect.zero)
         KeyboardHelper.defaultHelper.addDelegate(delegate: self)
         
-        searchButton.isHidden = true
-        searchButton.accessibilityIdentifier = "OverlayView.searchButton"
-        searchButton.alpha = 0
-        searchButton.setImage(#imageLiteral(resourceName: "icon_searchfor"), for: .normal)
-        searchButton.setImage(#imageLiteral(resourceName: "icon_searchfor"), for: .highlighted)
-        searchButton.backgroundColor = UIConstants.colors.background
-        searchButton.titleLabel?.font = UIConstants.fonts.searchButton
-        searchButton.backgroundColor = UIConstants.colors.background
-        setUpOverlayButton(button: searchButton)
-        searchButton.addTarget(self, action: #selector(didPressSearch), for: .touchUpInside)
-        addSubview(searchButton)
```

Comment:
Might help to add a comment like:         ```// Create actual search button and searchSuggestionCount additional buttons``` to explain why we're actually making searchButtonNumber + 1 buttons.

Reply:
Thanks for the tips. I choose to start with index 1 instead of 0.

Reply:
It's probably best for us to do ```for _ in 0..<self.searchSuggestionsCount``` rather than starting at index 1 (as that's what we use in the rest of the code base). After that change I can approve! Your use of ```for i in 1..``` later in this code is fine, as we're actually skipping the first element so it makes sense.

Let me know if you disagree!